cal/api/views.py

```diff
@@ -242,4 +242,4 @@ class TagDetailEventTimeSeries(APIView):
 
     def get(self, request, *args, **kw):
         tag = Tag.objects.get(user=self.request.user, id=self.kwargs['pk'])
-        return Response(tag.get_time_series(self.request.query_params.get('timezone')))
+        return Response(tag.get_time_series(self.request.query_params.get('timezone'), time_step=self.kwargs['time_step']))
```

Comment:
Is there a reason why you used self.kwargs instead of query_params? Is one better than the other? Should I change the 'timezone' to use self.kwargs? @MaxFangX I noticed you used query_params in other cases.